### PR TITLE
Fix UserAvatar UI Component

### PR DESCRIPTION
**Pull request by Angshu31**
Fix some style choices

### CODE REVIEW

**Comment by ealmiladi** [comment]
You can just set a default for className, then you don't need the ternary in-line.
```
className = "",
```

**Comment by Angshu31** [comment]
> You can just set a default for className, then you don't need the ternary in-line.
> 
> ```
> className = "",
> ```

Oh. What I did would get rid of the useless trailing space. Not that it matters much

File: kibbeh/src/ui/UserAvatar/SingleUser.tsx [hunk 1]
```diff
@@ -45,20 +45,16 @@ export const SingleUser: React.FC<AvatarProps> = ({
 }) => {
   return (
     <div
-      className={`relative inline-block ${className}`}
+      className={`relative inline-block${className ? ` ${className}` : ""}`}
```
Comment (benawad) [inline kibbeh/src/ui/UserAvatar/SingleUser.tsx]: I'll take the trailing space over ternary

**Comment by Angshu31** [comment]
I changed to trailing space

File: kibbeh/src/ui/UserAvatar/SingleUser.tsx (hunk 1 above)
Comment (Angshu31) [reply kibbeh/src/ui/UserAvatar/SingleUser.tsx]: Done

**Pull request merged**